components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java.orig

```diff
@@ -0,0 +1,2949 @@
+/*
```

Comment:
Is this an auto generated file from git merge tool? Do we need to maintain this? Please remove this if there is not specific need to maintain this.

Reply:
Addressed by :https://github.com/wso2/carbon-apimgt/pull/12044